model/src/main/java/oracle/kubernetes/weblogic/domain/model/Domain.java

```diff
@@ -480,41 +480,82 @@ public boolean equals(Object other) {
   }
 
   class Validator {
+    static final String DUPLICATE_SERVER_NAME_FOUND
+          = "More than one item under spec.managedServers in the domain resource has DNS-1123 name '%s'";
+    static final String DUPLICATE_CLUSTER_NAME_FOUND
+          = "More than one item under spec.clusters in the domain resource has DNS-1123 name '%s'";
+    static final String LOG_HOME_PATH_NOT_MOUNTED
+          = "No volume mount contains path for log home '%s', %s in the domain resource";
+    static final String BAD_VOLUME_MOUNT_PATH
+          = "The mount path '%s', in entry '%s' of domain resource additionalVolumeMounts, is not valid";
+    
```

Comment:
I18N these?

Reply:
What's our standard way for doing that?

Reply:
The Operator.properties resource bundle

Reply:
That's in the operator sub-module, to which this code has no access. Maybe we should merge the model project into the operator subproject?